+ndr/+test/+reader/+ced_smr/automatedTest.m

```diff
@@ -0,0 +1,96 @@
+function tests = automatedTest
+  tests = functiontests(localfunctions);
+end
+
+function setupOnce(test_case)
+  ndr_Init();
+end
+
+% Note that if we want to read the event channel e22, we should pass 22 as a channel argument, etc.
+%
+% Channel found (1/14): ai1 of type analog_in
+% Channel found (2/14): mk20 of type mark
+% Channel found (3/14): ai21 of type analog_in
+% Channel found (4/14): e22 of type event
+% Channel found (5/14): e23 of type event
+% Channel found (6/14): e24 of type event
+% Channel found (7/14): e25 of type event
+% Channel found (8/14): e26 of type event
+% Channel found (9/14): e27 of type event
+% Channel found (10/14): e28 of type event
+% Channel found (11/14): e29 of type event
+% Channel found (12/14): text30 of type text
+% Channel found (13/14): mk31 of type mark
+% Channel found (14/14): mk32 of type mark
+function test_readevents_epochsamples_native_marker(test_case)
```

Comment:
In general Intant reader doesn't seem to follow the comments under the `readevents_epochsamples_native()` class in `base.n`.
E.g., on line 86&87 - `t` in `verifyEqual(test_case, data, t);` should be an array of `1`s.

And it should be a single `data` array, not [`timesptamps`, `data`].